### ISSUE TITLE
params dont get added when created in the deleteFile call

### ISSUE THREAD

**Issue by kangarolf** [FineUploader/fine-uploader#1488]
deleteFile: {
    enabled: true,
    forceConfirm: false,
    method: 'POST',
    endpoint: 'myendpoint',
    params: {tempID: 'MYFIXEDID'}

}

When watching the query no tempID is sent.

**Comment by rnicholus** [comment]
Present in at least 5.3.2, likely other versions as well. Will investigate when this regression occurred soon.

**Comment by rnicholus** [comment]
Appears to only be an issue for the traditional endpoint uploader (not S3/Azure).

**Comment by rnicholus** [comment]
After closer inspection, I am _not_ able to reproduce this issue. I became suspicious when I remembered that there are [_extensive_ unit tests](https://github.com/FineUploader/fine-uploader/blob/5.3.2/test/unit/delete-file.js) covering this feature. In my own development setup, I was actually calling the `setDeleteFileParams` method in a callback, which made it appear as if the `deleteFile.params` option was being ignored by Fine Uploader, until I looked at my code closer. I suspect you might be in a similar situation.
